docs/content/stable/yugabyte-platform/administer-yugabyte-platform/back-up-restore-yba.md

```diff
@@ -0,0 +1,136 @@
+1. Enter a name for the folder where you want to store the backups.
+
+    If you are backing up more than one YugabyteDB Anywhere instance to the same storage, provide a name that uniquely identifies the instance you are backing up. For example, `yba_104.72.167.2_backups`.
+
+1. Enter the backup frequency in minutes. More frequent backups provide more recovery points but may impact performance.
+1. Click **Apply Changes**.
+
+Click **Remove** to disable periodic backups (this stops automatic backups but does not delete existing backup files).
+
+## Back up manually
+
+You can manually back up your YugabyteDB Anywhere installation. Manual backups can include Prometheus data and YugabyteDB releases, in addition to your YugabyteDB Anywhere settings and metadata.
+
+Manual backups are stored locally on the node that hosts your installation.
+
+Perform a manual backup:
+
+- Before upgrading YugabyteDB Anywhere.
+- When migrating YugabyteDB Anywhere to a different host.
+- When you need to include Prometheus metrics data.
+- When you need to include release archives.
+- For ad-hoc backups outside your periodic backup schedule.
+
+To create a one-time backup:
+
+1. Navigate to **Admin>Platform HA and Backups** and select **Automated Platform Backups**.
+1. Click **One-Time Platform Export**.
+1. Select the data you want to back up.
+
+    - **Platform Metadata**: YugabyteDB Anywhere installation settings and metadata.
+    - **YugabyteDB Releases**: Imported [YugabyteDB releases](../../manage-deployments/ybdb-releases/).
+    - **Universe Metrics (Prometheus)** — Prometheus metrics data for your universes.
+
+1. Specify the destination for the backup, as a full path. For example, `/opt/yugabyte/yba_backups` or `/tmp/yba_export`.
+
+    Ensure the directory exists and the `yugabyte` user has write permissions for the location.
+
+    The backup is saved as a `.tar.gz` file with a timestamped name.
+
+1. Click **Export**.
+
+    You can monitor progress of the backup on the **Tasks** page.
+
+## Restore YugabyteDB Anywhere
+
+When doing a restore, YugabyteDB Anywhere performs the following checks:
+
+- Existing universes. By default, due to the possibility of data loss, you can only do a restore if your instance is not managing any universes. You can override this by setting the **Allow YBA Restore With Universes** Global Runtime Configuration option (config key `yb.yba_backup.allow_restore_with_universes`) to true.
+
+- The selected backup is not older than one day. By default you cannot restore from backups older than one day, as the backup may be inconsistent if you performed management operations after the backup was taken. You can override this by setting the **Allow YBA Restore With Old Backup** Global Runtime Configuration option (config key `yb.yba_backup.allow_restore_with_old_backup`) to true.
+
+- [High availability](../high-availability/) is not running.
```

Comment:
I would say, make this the first bullet, followed by the other two because it is then clear how to override the settings with the usual - Refer to [Manage runtime configuration settings](https://deploy-preview-29557--infallible-bardeen-164bc9.netlify.app/stable/yugabyte-platform/administer-yugabyte-platform/manage-runtime-config/).